keys/tyler.beck.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGXfWGn1zv/GW5d6+teJW2y2Z5xZCrcHuf3xDGX55JvW tyler.beck@goodeggs.com
```

Comment:
Interesting. I think @tvelich also has this type of key that is a bit different from the other ones. But it looks reasonable to me.